app/models/changeset.rb

```diff
@@ -1,9 +1,7 @@
 # frozen_string_literal: true
 class Changeset
-  attr_reader :project, :repo, :previous_commit, :reference # used by plugins
-
-  BRANCH_TAGS = ["master", "develop"].freeze
-  ATTRIBUTE_TABS = %w[files commits pull_requests risks jira_issues].freeze
+  ATTRIBUTE_TABS = ["files", "commits", "pull_requests", "risks", "jira_issues"].freeze
+  MAIN_BRANCHES = ["master", "develop", "staging", "production"].freeze
```

Comment:
eww. there has to be a better way...

Reply:
catches the 99% ... good enough 🤷‍♂

Reply:
... also it's just an optimization to avoid lookups, so does not matter if it misses a few